app/middlewares/userTokenMiddleware.js

```diff
@@ -0,0 +1,6 @@
+const jwt = require('../services/jwt');
+
+exports.updateReq = (req, res, next) => {
```

Comment:
Why this name ?

Reply:
I'm bad with names :cry: 
UpdateReqHeaders perhaps?

Reply:
But why do you need this middleware? The validation middleware should do this already